proto/server.proto

```diff
@@ -33,4 +33,5 @@ message InfoRes {
   string client_config_dns_search_domain = 14;
   int32 client_config_mtu = 15;
   proto.BuildInfo build_info = 16;
+  int32 wireguard_mtu = 17;
```

Comment:
- Since we also omit the 'wireguard' prefix for `host` and `port` I'd rather leave it away as well for the `mtu`.
- Not all files have been regenerated; after running the codegen, `website/src/sdk/server_pb.ts` was dirty.

Reply:
> Not all files have been regenerated; after running the codegen, website/src/sdk/server_pb.ts was dirty.

Interesting, I thought I did that 🤔